src/source.c

```diff
@@ -1312,6 +1315,11 @@ static char *builtin_masters_update_info[] = {
 	// added 2003-11-18
 	"ADD CODS master://cod01.activision.com activision.com",
 
+	// added 2017-04-09
```

Comment:
This is pure bikeshedding on my own, but can you move these additions on top of the list ADD list (line 1128), so the more recent additions are sorted first? and please include the qtracker ones with them. It's not a problem if all the COD games are not together in this list, it's more helpful, on a maintainer point of view, to be able to find at first glance where are the oldest masters.